chainercv/transforms/image/pca_lighting.py

```diff
@@ -0,0 +1,48 @@
+import numpy
+
+
+def pca_lighting(img, sigma, eigen_value=None, eigen_vector=None):
+    """Alter the intensities of input image using PCA.
+
+    This is used in training of AlexNet [Krizhevsky]_.
+
+    .. [Krizhevsky] Alex Krizhevsky, Ilya Sutskever, Geoffrey E. Hinton. \
+    ImageNet Classification with Deep Convolutional Neural Networks. \
+    NIPS 2012."ImageNet Classification with Deep Convolutional Neural
+    Networks" [NIPS 2012].
```

Comment:
`"ImageNet..." [NIPS 2012]. `
can be eliminated.

Just you know, you can check the processed html documentation by `cd docs; make html`.